src/components/Timestamp.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+function Timestamp(props) {
+  let {timestamp} = props;
```

Comment:
Here, it's better practice if you use "const" instead of "let" when doing object destructuring